-- let_err3 --
let y = 2
z: y
x: {
  // redeclared in the same block (y has been declared in the top block)
  y: 2
}
-- exp --
error: can not have both alias and field with name y in the same scope
